cascade/cascade.py

```diff
@@ -1246,37 +676,20 @@ def distribute_global_resources(
     :param azuretable.TableService table_client: table client
     :param str ipaddress: ip address
     """
-    # set torrent session port listen
-    if _ENABLE_P2P:
-        global _TORRENT_SESSION
-        # create torrent session
-        logger.info('creating torrent session on {}:{}'.format(
-            ipaddress, _DEFAULT_PORT_BEGIN))
-        _TORRENT_SESSION = libtorrent.session()
-        _TORRENT_SESSION.listen_on(_DEFAULT_PORT_BEGIN, _DEFAULT_PORT_END)
-        _TORRENT_SESSION.stop_lsd()
-        _TORRENT_SESSION.stop_upnp()
-        _TORRENT_SESSION.stop_natpmp()
-        # bootstrap dht nodes
-        bootstrap_dht_nodes(loop, table_client, ipaddress, 0)
-        _TORRENT_SESSION.start_dht()
     # get globalresources from table
     try:
         entities = table_client.query_entities(
             _STORAGE_CONTAINERS['table_globalresources'],
             filter='PartitionKey eq \'{}\''.format(_PARTITION_KEY))
     except azure.common.AzureMissingResourceHttpError:
-        entities = None
+        entities = []
     nentities = 0
-    # check torrent info table for resource
-    if entities is not None:
-        for ent in entities:
+    for ent in entities:
+        resource = ent['Resource']
+        grtype, _ = get_container_image_name_from_resource(resource)
+        if grtype == _CONTAINER_MODE.name.lower():
             nentities += 1
-            if _ENABLE_P2P:
-                _check_resource_has_torrent(
-                    blob_client, table_client, ent['Resource'])
-            else:
-                _DIRECTDL_QUEUE.put(ent['Resource'])
+            _DIRECTDL_QUEUE.put(resource)
```

Comment:
can you add an else here and log that you're skipping a resource not matched to the container mode?